libvast/vast/system/actors.hpp

```diff
@@ -387,4 +400,8 @@ CAF_BEGIN_TYPE_ID_BLOCK(vast_actors, caf::id_block::vast_atoms::end)
 
 CAF_END_TYPE_ID_BLOCK(vast_actors)
 
+using vast_uuid_synopsis_map = std::map<vast::uuid, vast::partition_synopsis>;
+CAF_ALLOW_UNSAFE_MESSAGE_TYPE(std::shared_ptr<vast_uuid_synopsis_map>)
+CAF_ALLOW_UNSAFE_MESSAGE_TYPE(std::shared_ptr<vast::partition_synopsis>)
```

Comment:
I assume these don't get type ids because they should never be sent over the wire? A small comment would go a long way.

I'm also not sure whether this file is the correct place for these statements. Shouldn't we think of this as implementation detail of the synopsis / meta index?

Reply:
I think they need to be here since they're part of the interface for the `meta_index_actor`, so anyone who wants to send a message to the meta index needs to see these declarations.